client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { useState, useEffect, useRef } from 'react';
-import { useTranslation } from 'react-i18next';
+import { useTranslation, withTranslation } from 'react-i18next';
```

Comment:
I can't find the reason to invoke `withTranslation` function, after I have read the docs and looked at its [code](https://github.com/i18next/react-i18next/blob/0a4681e428c888fe986bcc0109eb19eab6ff2eb3/src/withTranslation.js#L5).

Can you walk me through the need for us to invoke it?

Reply:
I thought that screen readers don't get translated strings, so we need to invoke the function, but thinking more about it, this doesn't make sense.